### PR TITLE
Add Tools directory and files to Feat/dwin nextgen support

### PR DESCRIPTION
**Pull request by Thinkersbluff**
I forked this repository, to help me learn how to work with these tools on a local copy of my own, and as requested in the extui ReadMe.

UPDATE (27 April 2022):
I started this PR started as a "practice-run", by proposing that the DGUS Tools and How-To info be updated on the extui branch.  
It then continued to evolve, by accident  (oops!), when I continued to push my other changes to the same branch on my repository, while I finished updating the work that InsanityAutomation had started. 
At this point, the changes bundled under this PR are ready to consider for a squash-merge back into the main extui branch, with the possible exception that the wonderfully detailed "How To DGUS" knowledge has not been retained and should not be lost.

I recommend that we capture and move that information into the WIKI as its own dedicated section, and that we continue to work on updating and improving it, to lower the barrier for future developers and maintainers of this UI firmware.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,154 +1,23 @@
 # CR-6 Touchscreen software
-Attempt to extend the CR-6 touch screen software. This repository belongs to the [forked Marlin firmware repository](https://github.com/CR6Community/Marlin/).
+Attempt to extend the CR-6 touch screen software to operate with newer display hardware being sold with CR6 machines as of approximately Dec 2021.
```
Comment (Sebazzz) [inline README.md]: We might want to clarify here that the firmware is also still valid for "old models".

File: README.md [hunk 2]
```diff
@@ -1,154 +1,23 @@
...
+This repository belongs to Thinkersbluff and is forked from [the CR6Community Touchscreen Firmware repository](https://github.com/CR6Community/CR-6-touchscreen).
```
Comment (Sebazzz) [inline README.md]: Let's remove this before merging back 🤓

File: README.md [hunk 3]
```diff
@@ -1,154 +1,23 @@
...
 
 ## Downloads
 
-Please find official releases in the [Releases section](https://github.com/CR6Community/CR-6-touchscreen/releases). Take the release which belongs to the particular firmware you are going to flash.
-
-Ensure you take the right assets: the `CR-6-touchscreen-[date].zip`. You should not download the `Source code` archive if you are downloading with the purpose of flashing your LCD.
+This project is not yet ready to release a stable version of the updated display firmware.
```
Comment (Sebazzz) [inline README.md]: I propose we merge this back to master as this is the only way forward, given the binary nature of the project,.

File: doc/build/DRAFT_21 Apr 2022/CR-6 kernel upgrade/README.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+To Flash		Copy this file to DWIN_SET:
+v3.5   		T5L_UI_DGUS2_V35_20200730.BIN
+v4.0   		T5L_UI_DGSU2_V40_Fast boot_20210917.BIN 
+v4.5			T5L_UI_DGUS2_V45_20220105.BIN
+v4.7			T5L_UI_DGUS2_V47_20220312.BIN
+
+NOTE: 
+Based on CF User feedback, we believe the following to be true:
+ - Legacy display hardware (Dec 2021 or earlier) does not run on v4.5 or v4.7
```
Comment (Sebazzz) [inline doc/build/DRAFT_21 Apr 2022/CR-6 kernel upgrade/README.txt]: We need to refine this to make sure and also include instructions on how to find the current version of the display firmware (the empty DWIN_SET folder I believe)

Comment (Sebazzz) [reply doc/build/DRAFT_21 Apr 2022/CR-6 kernel upgrade/README.txt]: And of course incorporate this back into the main README.

**Comment by Sebazzz** [comment]
LGTM so far!

Let's refine the README and flashing instructions a bit, remove the DRAFT files (so the repository size is less inflated - let's squash-commit-merge this so those files won't exist in the end result), and then I think we can offer these files as part of a 6.1b beta release.

**Comment by Thinkersbluff** [comment]
So this is part of what I mean when I say I am not very GitHub literate.  

I did not realize that continuing to commit changes to the same branch on my fork would result in a continuing evolution of the PR.

I have removed the DRAFT folders.

I am now refining the README.MD file in the RC folder.
Are there specific points you would like me to add/delete/modify?

**Comment by Sebazzz** [comment]
The most important thing to clarify is the DWIN OS upgrade. 

As for the rest, check my remarks in the "files" tab.

**Review by Sebazzz — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Thinkersbluff) [reply README.md]: There is a lot of detailed info in the current extui baseline README about how to contribute changes to this firmware.
I found it very helpful when I was working with it, but I deleted it completely from the README in the feat branch.
If we merge this branch back as-is, will all of that good info disappear?  I was hoping to keep it but update it to point to the new tools & to add any new instructions that may be required, but we could also create a separate document "For Developers Only" and plan to refine that in a follow-on PR.

**Comment by Thinkersbluff** [comment]
I believe this conflicting part of the PR has been overtaken by events.
As long as the latest TFT files are kept in the merge, we will be ok ignoring these ones.

**Comment by Sebazzz** [comment]
I see what the problem with merging is. This PR wants to merge to the feature branch instead of extui, but we should merge to extui instead.

**Comment by Sebazzz** [comment]
Changed to extui.

**Review by Sebazzz — APPROVED** [review]

**Pull request merged**